### PR TITLE
Firebolt fix

### PR DESCRIPTION
**Pull request by tchalvak**
Going to have to do more review of the behavior of skill_mods, because something bad still seems to be happening in various places.

This PR also includes the code to make Admins get all skills in the game (at least on the player profile, the skills list skills will follow when I can, I'll create an issue for that).

### CODE REVIEW

File: deploy/www/player.php [hunk 1]
```diff
@@ -75,19 +76,24 @@
 		$level_category       = level_category($player_info['level']);
 		$gurl = $gravatar_url = generate_gravatar_url($target_player_obj);
 
-		if ($char_id && !$attack_error && !$self) { // They're not dead or otherwise unattackable.
+		if ($viewing_player_obj->id() && !$attack_error && !$self) { // They're not dead or otherwise unattackable.
```
Comment (Beagle) [inline deploy/www/player.php]: @tchalvak  Is it likely that $viewing_player_obj->id() will throw a null pointer exception in cases where $char_id would have been blank? This seems like a potentially dangerous change. Consider just testing if $viewing_player_obj is not null?

File: deploy/www/player.php [hunk 2]
```diff
@@ -109,7 +115,7 @@
 
 			if ($viewer_clan) {
 				$same_clan = ($clan->getID() == $viewer_clan->getID());
-				$display_clan_options = ($username && !$self && $same_clan && is_clan_leader($viewing_player_obj->vo->player_id));
+				$display_clan_options = ($viewing_player_obj->name() && !$self && $same_clan && is_clan_leader($viewing_player_obj->vo->player_id));
```
Comment (Beagle) [inline deploy/www/player.php]: @tchalvak Same concern as above. My assumption is that a test for $username is a test for existence. Your change may result in a null pointer exception in those cases. Please review.

File: deploy/www/player.php (hunk 1 above)
Comment (tchalvak) [inline deploy/www/player.php]: Wasn't really practical with $char = new Player($id); so I just implemented Player::find($id);

File: deploy/www/player.php (hunk 2 above)
Comment (tchalvak) [inline deploy/www/player.php]: Fixed in commit to be pushed soon.

**Comment by Beagle** [comment]
:shipit:

**Comment by tchalvak** [comment]
cruise-ship It

**Comment by tchalvak** [comment]
wvl6gzz

...It

**Pull request merged**